product-list/index.html

```diff
@@ -191,7 +193,7 @@ <h2>Iniciar sesión</h2>
             fill="#352B2B" />
         </svg>
       </span>
-      <a class="indicator__link indicator__link--border" href="#">Lista de productos</a>
+      <a class="indicator__link indicator__link--border" href="#">Productos</a>
```

Comment:
This href doesn't have the link to the products yet, this is on the products page, but I also saw that it is found like this on the other pages so it would be nice to put it.